app/assets/stylesheets/components/styled-box.scss

```diff
@@ -168,6 +174,10 @@
     background-size: cover;
     background-repeat: no-repeat;
 
+    display: flex;
```

Comment:
A good opportunity exists to refactor this to follow the property sort order rule here if you want: https://github.com/sasstools/sass-lint/blob/develop/docs/rules/property-sort-order.md

Reply:
Done! I've been trying to keep this one in mind, so good practice to go in and update some prior work.